src/processes.c

```diff
@@ -346,6 +346,58 @@ int getargs(void *processBuffer, int bufferLen, char *argsBuffer, int argsLen);
 static ts_t *taskstats_handle;
 #endif
 
+enum {
+  FAM_PROC_VMEM_SIZE = 0,
+  FAM_PROC_VMEM_RSS,
+  FAM_PROC_VMEM_DATA,
+  FAM_PROC_VMEM_CODE,
+  FAM_PROC_VMEM_STACK,
+  FAM_PROC_CPU_USER,
+  FAM_PROC_CPU_SYSTEM,
+  FAM_PROC_NUM_PROCESSS,
+  FAM_PROC_NUM_THREADS,
+  FAM_PROC_VMEM_MINFLT,
+  FAM_PROC_VMEM_MAJFLT,
+  FAM_PROC_IO_RCHAR,
+  FAM_PROC_IO_WCHAR,
+  FAM_PROC_IO_SYSCR,
+  FAM_PROC_IO_SYSCW,
+  FAM_PROC_IO_DISKR,
+  FAM_PROC_IO_DISKW,
+  FAM_PROC_FILE_HANDLES,
+  FAM_PROC_FILE_HANDLES_MAPPED,
+  FAM_PROC_CTX_VOLUNTARY,
+  FAM_PROC_CTX_INVOLUNTARY,
+  FAM_PROC_DELAY_CPU,
+  FAM_PROC_DELAY_BLKIO,
+  FAM_PROC_DELAY_SWAPIN,
+  FAM_PROC_DELAY_FREEPAGES,
```

Comment:
I think a bunch of these should be combined, so that we end up with the following families:

* vmem
* cpu (usage/utilization)
* processes
* threads
* memory faults
* I/O
* file handles
* context switches
* delay

I think a good rule of thumb is: if metrics use the same unit and could conceivably be aggregated together, then they should be in the same metric family.